docker-compose.yml

```diff
@@ -60,9 +66,13 @@ services:
       POSTGRES_DB: kolibri-studio
     volumes:
       - pgdata:/var/lib/postgresql/data/pgdata
+    networks:
+      - studio-network
```

Comment:
Nice! Has this always been required, or only in recent versions of docker-compose?

Reply:
Good question. I thought it was but apparently it's not. I think I'll remove this as it appears docker-compose will do this automatically. Most importantly, it does appear that the automatically created networks from docker-compose are [isolated](https://stackoverflow.com/questions/38088279/communication-between-multiple-docker-compose-projects) which is what I definitely did not think was the case.